commands/priority_test.go

```diff
@@ -49,3 +49,51 @@ func Test_getTemplateStoreURL(t *testing.T) {
 		})
 	}
 }
+
+func Test_getOverrideNamespace(t *testing.T) {
+	const defaultNamespace = defaultFunctionNamespace
+
+	tests := []struct {
+		stackNamespace    string
+		flagNamespace     string
+		expectedNamespace string
+	}{
+		// Test cases
+		{
+			stackNamespace:    "",
+			flagNamespace:     "",
+			expectedNamespace: defaultNamespace,
+		},
+
+		{
+			stackNamespace:    "openfaas-fn",
+			flagNamespace:     "default",
```

Comment:
The default is `openfaas-fn` so I find this example rather confusing.

Was there a reason why you didn't use the samples given by @LucasRoesler ? https://github.com/openfaas/faas-cli/issues/742#issuecomment-625746405

in place of `expectedNamespace` please use `want`

`Namespace` as a suffix is probably not needed in the test table at all, it could be simplified to:

```
		stack    string
 		flag     string
 		want string
```

```
			t.Errorf("Expected Namespace: %s\nGot: %s\n", testCase.expectedNamespace, functionNamespace)
```

To:

```
			t.Errorf("want %s, but got %s", case)
```

I'd also like to see a named test-cases as follows:

```
stack: ""
flag: ""
want: defaultNs
scenario: "no value in stack or flag"
```


```
stack: ""
flag: "staging"
want: "staging"
scenario: "not set in stack.yml, but set as flag"
```

Then in each iteration of the loop, you should run a sub-test as we do elsewhere, this will output the name of the test scenario in the logs, in the case of a failure.

```
        t.Run(testCase.scenario, func(t *testing.T) {

       }

```